Formula/acmetool.rb

```diff
@@ -18,7 +18,10 @@ class Acmetool < Formula
   deprecate! date: "2020-06-01"
 
   depends_on "go" => :build
-  uses_from_macos "libpcap"
+
+  on_linux do
+    depends_on "libcap"
```

Comment:
depends_on "libpcap"

Reply:
This is what I've been stating throughout the last few PR's, but am happy to repeat again. It's `libcap` and it's available on Linux. As stated above, `brew test-bot` has run through on my Linux machine producing the correct build result:

```
mikesvps@ubuntu:~$ readelf -d $(which acmetool)

Dynamic section at offset 0xa76dd8 contains 28 entries:
  Tag        Type                         Name/Value
 0x0000000000000001 (NEEDED)             Shared library: [libcap.so.2]
 0x0000000000000001 (NEEDED)             Shared library: [libpthread.so.0]
 0x0000000000000001 (NEEDED)             Shared library: [libc.so.6]
 0x000000000000000f (RPATH)              Library rpath: [/home/linuxbrew/.linuxbrew/Cellar/acmetool/0.0.67_1/lib:/home/linuxbrew/.linuxbrew/lib:/home/linuxbrew/.linuxbrew/opt/libcap/lib]
```
This was correct last time this formula was bottled at https://github.com/Homebrew/homebrew-core/commit/d293c2d0. The problem there was the uncertainty due to the use of HEAD dependencies (binary will differ from time to time and may eventually break some day), but the bottle itself was functional. All commits after that has made the formula become defective/produce unwanted results. I remember this had to do with coping with `systemd` on Linux.

Reply:
Ho right, sorry, I did not follow that part of the discussion.

In this case, and as this formula is deprecated, we will not build any new bottles for it on Linux.
So just drop the dependency completely.

Sorry for the back and forth

Reply:
But there might still be users trying to build this formula by hand (given that there are a lot more users using Linuxbrew on custom prefixes, including myself, than on macOS). In that case it would be better to have this formula done correctly.

Reply:
Plus, I'm still very curious about the error in CI here. How come this doesn't happen on my machines, neither on macOS nor on Linux?